src/EditorFeatures/Core/Implementation/FindReferences/DefinitionItem.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Completion;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.FindReferences
+{
+    /// <summary>
+    /// Information about a symbol's definition that can be displayed in an editor
+    /// and used for navigation.
+    /// </summary>
+    internal sealed class DefinitionItem
+    {
+        /// <summary>
+        /// Descriptive tags from <see cref="CompletionTags"/>. These tags may influence how the 
+        /// item is displayed.
+        /// </summary>
+        public ImmutableArray<string> Tags { get; }
```

Comment:
Tags will affect things like the glyph that is displayed.  This is the same as completion.